cms/static/cms/js/require-config.js

```diff
@@ -27,7 +27,8 @@
             "mustache": "js/vendor/mustache",
             "codemirror": "js/vendor/codemirror-compressed",
             "codemirror/stex": "js/vendor/CodeMirror/stex",
-            "jquery": "js/vendor/jquery.min",
+            "jquery": "common/js/vendor/jquery",
+            "jquery-migrate": "common/js/vendor/jquery-migrate",
             "jquery.ui": "js/vendor/jquery-ui.min",
```

Comment:
@bjacobel Do you think we need to test all of these JQuery plugins, or should JQuery Migrate make that unnecessary? Should we be upgrading any of these to newer versions (not as part of this PR)?